### DISCUSSION TITLE
Combine RGB image with Alpha channel?

### DISCUSSION THREAD

**Discussion by Isak-Andersson** [ImageMagick/ImageMagick#3762] [Help]
Hi. I'm trying to combine this:

![RGB](https://user-images.githubusercontent.com/78758898/120078829-9830a000-c0b1-11eb-874a-f6cae367fabe.png)

With this:

![A](https://user-images.githubusercontent.com/78758898/120078832-98c93680-c0b1-11eb-8419-99af6f0b2d91.png)

To make this:

![RGBA](https://user-images.githubusercontent.com/78758898/120078830-98c93680-c0b1-11eb-81d3-a5b6ba0c0e8a.png)

Basically, I just want to use the RGB channels of "RGB.png" and the alpha channel of "A.png" and save it to "RGBA.png". Can it be done? Thanks.

**Comment by dlemstra** [comment]
You can do that with the following command:

`magick RGB.png A.png -compose copyalpha -composite RGBA.png`

**Reply by Isak-Andersson** [reply]
Excellent! Worked like a charm. This will save me a ton of time. Thanks a lot!
